backend/api/models.py

```diff
@@ -1,7 +1,5 @@
-import uuid
```

Comment:
why was this removed? If I run `python manage.py`, I get an error because `uuid.uuid4` cannot be found :)

Reply:
Actually uuid.uuid4 should not be used anymore

Reply:
Still, I get the error upon launch (`python manage.py` does not work as well)?

Reply:
Maybe you still got an old database scheme that somewhere uses uuid4?

Reply:
that did the trick, thank you!